docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst

```diff
@@ -0,0 +1,64 @@
+.. _guides-experiment-dashboard:
+
+Organizing Experiments in a Dashboard
+=====================================
+
+|Youtube Video dashboard|
+
+Neptune is a browser-enabled app that lets you visualize and browse experiments.
+
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/experiment_general_view.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/experiment_general_view.png
+      :alt: Experiments view
+
+The **Experiments** space displays all the experiments in a specific Project in table form.
+
+There are several ways to organize your experiments.
+
+Using tags
+----------
+
+You can create tag(s), which you assign to experiments. Later, you can quickly filter by these tags.
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/tag_chooser.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/tag_chooser.png
+      :alt: Tag chooser
+      :width: 250
+
+Using dashboard views
+---------------------
+
+You can create a custom view of the dashboard. For example, you can filter rows by parameter or metric values and select a subset of useful columns.
+
+Then you can save that view and quickly return to it by selecting it from the list of views.
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/view_list.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/view_list.png
+      :alt: View list
+      :width: 400
+
+Customizing columns
+-------------------
+
+You can configure several data types logged to Neptune so that they are displayed as columns in the dashboard. They are metrics, text logs, properties and parameters. However, **all** data can be seen in the **single** experiment view.
```

Comment:
[*consider replacing*] 
> You can configure several data types logged to Neptune so that they are displayed as columns in the dashboard.

Almost all data logged to the experiment can be displayed as a dashboard column.